src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -101,15 +101,18 @@ const create = function(uiSchemaObj) {
  * }
  *
  */
-const createIdpButtons = (remediations) => {
-  const redirectIdpRemediations = remediations.filter(idp => idp.name === RemediationForms.REDIRECT_IDP);
+const createIdpButtons = (settings, appState) => {
```

Comment:
nit: Consider using destructured params here to cleanup this method signature.

```suggestion
const createIdpButtons = ({ settings, appState }) => {
```

And the callee:

```diff
-createIdpButtons(this.options.settings, this.options.appState);
+createIdpButtons(this.options);
```